pkg/ccl/backupccl/restore_online.go

```diff
@@ -150,37 +264,13 @@ func sendAddRemoteSSTWorker(
 	approxDataSize *int64,
 ) func(context.Context) error {
 	return func(ctx context.Context) error {
-		var toAdd []execinfrapb.RestoreFileSpec
-		var batchSize int64
-		var err error
-		const targetBatchSize = 440 << 20
-
-		flush := func(splitAt roachpb.Key, elidedPrefixType execinfrapb.ElidePrefix) error {
-			if len(toAdd) == 0 {
-				return nil
-			}
-
-			if len(splitAt) > 0 {
-				if err := sendSplitAt(ctx, execCtx, splitAt); err != nil {
-					log.Warningf(ctx, "failed to split during experimental restore: %v", err)
-				}
-			}
-
-			for _, file := range toAdd {
-				if err := sendRemoteAddSSTable(ctx, execCtx, file, elidedPrefixType, fromSystemTenant); err != nil {
-					return err
-				}
-			}
-			toAdd = nil
-			batchSize = 0
-			return nil
-		}
-
 		for entry := range restoreSpanEntriesCh {
-			firstSplitDone := false
+			log.Infof(ctx, "starting restore span %s with %d files", entry.Span, len(entry.Files))
```

Comment:
nit: this log line will be confusing as its in old key space

Reply:
I'll change it to say "backed up span %s"